cursorless-talon/.gitrepo

```diff
@@ -0,0 +1,12 @@
+; DO NOT EDIT (unless you know what you are doing)
+;
```

Comment:
So this is the file generated by `git-subrepo`.  Keeps track of all the information needed for pushing.  Pretty cool